clar2wasm/tests/wasm-generation/control_flow.rs

```diff
@@ -118,10 +101,14 @@ proptest! {
     #![proptest_config(super::runtime_config())]
 
     #[test]
-    fn begin((expr, expected_val) in begin_strategy()) {
-        crosscheck(
-            &expr,
-            Ok(Some(expected_val.into()))
-        );
+    fn begin((expr, expected_val, is_response_intermediary) in begin_strategy()) {
+        if is_response_intermediary{
+            crosscheck_compare_only(&expr);
```

Comment:
Why not `crosscheck`? We know exactly the expected answer is for now `Err(())`, and we will have a better testing of the error after the merge of the runtime error support.

Reply:
Fixed